cmd/kubeadm/app/phases/copycerts/copycerts.go

```diff
@@ -89,7 +89,8 @@ func UploadCerts(client clientset.Interface, cfg *kubeadmapi.InitConfiguration,
 	fmt.Printf("[upload-certs] Storing the certificates in Secret %q in the %q Namespace\n", kubeadmconstants.KubeadmCertsSecret, metav1.NamespaceSystem)
 	decodedKey, err := hex.DecodeString(key)
 	if err != nil {
-		return err
+		klog.Error("[upload-certs] Failed decoding certificate key.")
+		return errors.Wrap(err, "error decoding certificate key")
```

Comment:
isn't this sufficient?
`return errors.Wrap(err, "error decoding certificate key")`
the error should bubble up and be printed.

i think with this change it will be printed 2times - once with klog once from cobra.